test/test_sync.cpp

```diff
@@ -6154,7 +6154,9 @@ TEST(Sync_Dictionary)
     }
 }
 
-TEST(Sync_CollectionInMixed)
+constexpr bool SYNC_SUPPORTS_NESTED_COLLECTIONS = false;
+
+TEST_IF(Sync_CollectionInMixed, SYNC_SUPPORTS_NESTED_COLLECTIONS)
```

Comment:
Can we sync with the legacy sync server? 🤔

Reply:
Yes

Reply:
wow, how so?

Reply:
Well - the legacy server uses the same applier - but no conflict resolution is done.

Reply:
I see. So I guess last update wins.